openpype/lib/events.py

```diff
@@ -443,6 +450,51 @@ def emit_event(self, event):
         self._current_event = None
 
 
+class OrderedEventSystem(EventSystem):
```

Comment:
Shouldn't this use `QueuedEventSystem`?

Reply:
No, that's first in first out right? Or can you explicitly set an order number for what you register so something registered later can still register itself before or after others callbacks based on `order` value.

Reply:
Will be resolved with #6080